<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mobinstatus</class>
 <widget class="QWidget" name="mobinstatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytMbnStts">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytMbnStts
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytMbnStts.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmmndBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>153</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/cmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>152</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstBtn">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>10</y>
     <width>153</width>
     <height>93</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pwrBtn">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>10</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pwrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pwrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingBtn">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clrBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/clrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tempClrBtn">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tempClrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/tempClrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckBtn">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>376</y>
     <width>152</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clpPrxyPed">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>156</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clpPrxyPed
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnClpPrxy">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>187</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnClpPrxy
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upClpPrxy">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>219</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upClpPrxy
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dirPrxy">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>251</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dirPrxy
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnMicSw">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>280</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnMicSw
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upMicSw">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>308</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upMicSw
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedClp">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>182</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedClp
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnClp">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>209</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnClp
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upClp">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>235</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upClp
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="opnMicSwSnrf">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>263</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#opnMicSwSnrf
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clsMicSwSnrf">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>289</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#clsMicSwSnrf
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="snrfMot">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>313</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#snrfMot
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnMot">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>340</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnMot
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedParkSttsMbnBtn">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>156</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedParkSttsMbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pedPwrSttsMbnBtn">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>338</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pedPwrSttsMbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
